Source/Particles/WarpXParticleContainer.cpp

```diff
@@ -280,15 +280,6 @@ WarpXParticleContainer::DepositCurrent(WarpXParIter& pti,
         tilebox = amrex::coarsen(pti.tilebox(),ref_ratio);
     }
 
-#ifndef AMREX_USE_GPU
-    // Staggered tile boxes (different in each direction)
-    Box tbx = convert( tilebox, jx->ixType().toIntVect() );
-    Box tby = convert( tilebox, jy->ixType().toIntVect() );
-    Box tbz = convert( tilebox, jz->ixType().toIntVect() );
-#endif
-
-    tilebox.grow(ng_J);
```

Comment:
Without this line, `lo` below will be different and this all `j` array accesses on GPU are, too.

Reply:
This PR is indeed changing the value of `const Dim3 lo = lbound(tilebox);`.

However, it should not matter (up to machine precision errors), because we pass both
```
const Dim3 lo = lbound(tilebox);
const std::array<Real, 3>& xyzmin = WarpX::LowerCorner(tilebox, galilean_shift, depos_lev);
```
to the deposition kernel. As long as `lo` (integer position of the corner) and `xyzmin` (position of the corner in physical units) correspond to the same point, the current deposition kernel should do the right thing (independently of the number of guard cells used in `tilebox`).

At least this was my understanding ; but the checksum suggests that this is not the case. I'll have a closer look.